bgpd/bgp_table.h

```diff
@@ -219,7 +231,17 @@ static inline struct bgp_dest *bgp_route_next_until(struct bgp_dest *dest,
 static inline struct bgp_dest *bgp_node_get(struct bgp_table *const table,
 					    const struct prefix *p)
 {
-	return bgp_dest_from_rnode(route_node_get(table->route_table, p));
+	struct route_node *rn = route_node_get(table->route_table, p);
+
+	if (!rn->info) {
+		struct bgp_dest *dest = XCALLOC(MTYPE_BGP_NODE,
```

Comment:
So we are dropping `bgp_node_create()` entirely and moved the logic here, right?

Reply:
Yes.